src/features/profile/skills/show-skills/show-skills.tsx

```diff
@@ -0,0 +1,24 @@
+import { Box } from "@mui/material";
+import { Skill } from "core/profiles/types";
+import DeleteSkillModal from "../delete-skills/modal";
+import SkillCard from "./skill-card";
+
+const ShowSkills = ({ skills, isEditable }: ShowSkillsProps) => {
+  return (
+    <>
+      {skills.map((item: Skill) => (
+        <Box key={item.id}>
+          <SkillCard item={item} />
+          {isEditable && <DeleteSkillModal skillId={item.id} />}
+        </Box>
+      ))}
+    </>
+  );
+};
+
+interface ShowSkillsProps {
```

Comment:
uso local, nombres locales
```suggestion
interface Props {
```